Reduce doc bloat & edit LDAP external authentication user story (#2975)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-tls-for-secure-ldap.adoc

[id="Configuring_TLS_for_Secure_LDAP_{context}"]
= Configuring TLS for secure LDAP

> If you require {Project} to use TLS to establish a secure LDAP connection (LDAPS), obtain certificates used by the LDAP server you are connecting to and mark them as trusted on the base operating system of your {ProjectServer}.

Reviewer: ```suggestion
If {Project} uses TLS to establish a secure LDAP connection (LDAPS), you must obtain the CA certificates  of your LDAP server and add them to the trusted CA list on the base operating system of your {ProjectServer}.
```

> If your LDAP server uses a certificate chain with intermediate certificate authorities, all root and intermediate certificates in the chain must be trusted, so ensure you obtain all certificates.

Reviewer: ```suggestion
If your LDAP server uses a certificate chain with intermediate certificate authorities,  you must obtain all root and intermediate certificates and add them to the trusted CA list.
```

.Procedure

> . Obtain the certificate from the LDAP Server.

Reviewer: ```suggestion
. Obtain the CA certificate from the LDAP Server:
```
Lead-in sentences end with a colon.

> .. If you use Active Directory Certificate Services, export the Enterprise PKI CA Certificate using the Base-64 encoded X.509 format.

Author: That's alright, fixed.

ifndef::orcharhino[]
See https://access.redhat.com/solutions/1498773[How to configure Active Directory authentication with `TLS` on {Project}] for information on creating and exporting a CA certificate from an Active Directory server.
endif::[]

> .. Download the LDAP server certificate to a temporary location onto {ProjectServer}, such as `/tmp/_example.crt_`.

Reviewer: ```suggestion
.. Download the LDAP server certificate to a temporary location on the {ProjectServer}, such as `/tmp/_example.crt_`.
```

> You will remove the certificate when finished.

Reviewer: Not sure you need to mention this because the last step is to delete the downloaded certificate.

Author: I know what you mean, but I find it to be useful context for selecting the download location. No change.
